parm/ioda/bufr2ioda/bufr2ioda_adpupa_prepbufr.json

```diff
@@ -0,0 +1,11 @@
+{
+  "data_format"      : "prepbufr",
+  "data_type"        : "prepbufr",
+  "cycle_type"       : "gdas",
```

Comment:
cycle type should not be fixed, we need to support both GFS and GDAS cycles

Reply:
@CoryMartin-NOAA cycle_type has been changed to 'DUMP' from gdas, as used in satwind file. Thank you for catching this.